docs/changelog.txt

```diff
@@ -82,6 +82,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - `gui/autodump`: fort-mode keybinding: Ctrl-H (when ``armok`` tools are enabled in `gui/control-panel`)
 - Window behavior: if you have multiple DFHack tool windows open, scrolling the mouse wheel while over an unfocused window will focus it and raise it to the top
 - `stockpiles`: allow filtering creatures by tameability
+- Autonick: additional nicknames based on borrowwing animals, colours, gems and minerals added
```

Comment:
This section down here was for the 50.08-r2 release, which went out a few days ago. New changelog entries should go in the "Future" section.

since "Autonick" is a command name, surround it in single quotes so that it get properly hyperlinked in our docs. e.g.:

```
- `autonick`: additional nicknames based on...
```

is `borrowwing` supposed to be `borrowing` or `burrowing`?

(sorry, I know this  is a lot of notes for a one-line text change) I'm actually in the middle of a hotfix release right now, and the "Future" section has moved up another notch. before you make any further changes to this file, could you pull in recent updates from develop? There's a green button on the PR that says "Update branch". Click that.